### PR TITLE
Remove Apache Commons IO dependency and split into smaller methods

### PR DESCRIPTION
**Pull request by lsoares**

### CODE REVIEW

**Comment by lsoares** [comment]
Can you check now?

**Comment by lsoares** [comment]
Hi @KevinGilmore 
I think I did it correctly now.
I only see my commits in this PR now.
Sorry for the hassle.

File: jsoup/src/test/java/com/baeldung/jsoup/JsoupParserTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.baeldung.jsoup;
+
+import java.io.IOException;
+import static junit.framework.Assert.assertEquals;
```
Comment (KevinGilmore) [inline jsoup/src/test/java/com/baeldung/jsoup/JsoupParserTest.java]: The junit.framework package is deprecated in JUnit 4+.
Change these static imports to org.junit.Assert.assertEquals and org.junit.Assert.assertTrue

**Comment by lsoares** [comment]
done

**Pull request merged**